### ISSUE TITLE
Responsive media (video) container shows black borders

### ISSUE THREAD

**Issue by chriscarreck** [chriscarreck/gradstream#11]
Hey Edvinas,
Keen to get your thoughts on this one. I notice that at certain widths, the responice media frame that contains the video stretches to fit the width of the containing div. The problem is the video itself doesnt resize, so you end up with massive black borders at the edges as per the image below.

Do you think it is ok to leave like this? I guess we can stretch the video itself it would look weird. Is there any way to make the media box fit the width of the video? I guess then there would be a lot of white space either side, but would this look better than the black space it currently has? 
The exmaple below is from the home screen:

![image](https://cloud.githubusercontent.com/assets/7758549/9422542/79c3b56c-48da-11e5-8600-977eb92f1aa9.png)

**Comment by EdvinasDaugirdas** [comment]
Yeah Chris, I understand what you're trying to say. Unfortunately, there is nothing I can do with this matter, that is just how the player works. The video itself is an attribute (source), not an object, so there is no way I can get the "width" or "height" of an attribute. And to tell you the truth, I think the borders are alright, it does not seem bad, these things happen.

**Comment by chriscarreck** [comment]
ok cool no probs, i'll just leave this as is, i dont think its a major thing either. Cheers
